code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -108,6 +108,10 @@
 	if(HAS_TRAIT(src, TRAIT_NOGUNS))
 		to_chat(src, "<span class='warning'>You can't bring yourself to use a ranged weapon!</span>")
 		return FALSE
+	if(G.trigger_guard == TRIGGER_GUARD_NORMAL)
+		if(HAS_TRAIT(src, TRAIT_TINY))
+			to_chat(src, "<span class='warning'>You're too small to reach the trigger!'</span>")
+			return FALSE
```

Comment:
Trigger guard was checked for `TRIGGER_GUARD_NORMAL` 3 lines prior, you can piggyback on that instead of checking it a second time

Reply:
how'd I get the message though?